book/website/communication/open/education.md

```diff
@@ -0,0 +1,82 @@
+(cm-open-education)=
+# Open Education 
+
+(cm-open-education-what)=
+## What is Open Education
+Open Education is a collective term that encompasses educational resources, tools and practices that can be freely used in the digital environment without legal, financial or technical barriers. 
+Open Education can also include [Open Pedagogy](http://openpedagogy.org/open-pedagogy/) which also focuses on the conversation with each other to develop educational practices and structures. 
+In open educational practices there is more space for [participation of learners in the knowledge creation and sharing](https://opencontent.org/blog/archives/2975), empowering learners and stimulating peer-learning and colllaboration.  
+
+(cm-open-education-why)=
+## Why Open Education?
+
+Open Education is important as it:
+
+- Facilitates broader and more equal access to access to knowledge and knowledge generation.
+Open Education is rooted in the human right to access high-quality education.
+- Is more easy to use: 
+    - Easier to build upon and correct the materials, which facilitates customisation of your education curriculum.
+    - Easier to collaborate, participate and share resources.
+- Reduces duplication of effort: If an educator needs materials for teaching and such materials already exist openly, then they need not make their own from scratch, thereby saving time.
+- Is generally more accessible due to open formats compared to proprietary formats in which files cannot be changed. 
```

Comment:
"due to open" -> "due to the use of open"

Reply:
thank you!